docs/conf.py

```diff
@@ -0,0 +1,48 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = "outpack-py"
+copyright = "2024, Rich FitzJohn"
+author = "Rich FitzJohn"
+release = "0.0.1"
```

Comment:
this would ideally reflect the number in `src/outpack/__about__.py`; that can be imported. Similarly if we could get the project slurped in we'd be able to copy this file around between projects easily

Reply:
done